### PR TITLE
WT-6596 Increase cache for timestamp abort test.

### PR DESCRIPTION
**Pull request by ddanderson**
Ensure threads' key spaces are truly separate.
Show the wiredtiger_open configuration on output.

### CODE REVIEW

**Comment by ddanderson** [comment]
@sueloverso, can you please review?

File: test/csuite/timestamp_abort/main.c [hunk 1]
```diff
@@ -85,21 +85,27 @@ static volatile uint64_t global_ts = 1;
...
 #define ENV_CONFIG_TXNSYNC \
     ENV_CONFIG_DEF         \
     ",transaction_sync=(enabled,method=none)"
 #define ENV_CONFIG_REC "log=(archive=false,recover=on)"
 
+/*
+ * A minimum width of 10, along with zero filling, means that all the keys sort according to their
+ * integer value, making each thread's key space distinct.
+ */
+#define KEY_FORMAT ("%010" PRIu64)
```
Comment (sueloverso) [inline test/csuite/timestamp_abort/main.c:107]: Hmm, `random_abort` uses a similar key space. Can you check if it also has the problem and fix it there if needed?

File: test/csuite/timestamp_abort/main.c [hunk 2]
```diff
@@ -438,18 +444,18 @@ run_workload(uint32_t nth)
...
+     * default, a leaf page grows to 32K in size before it splits and the thread begins to fill
+     * another page. We'll budget for 10 full size leaf pages per thread in the cache plus a little
+     * extra in the total for overhead.
      *
      * The configuration sets the eviction update and dirty targets at 20% so that on average, each
-     * thread can have a dirty page before eviction threads kick in. On the other side, the eviction
-     * update and dirty triggers are 95%, so application threads aren't involved in eviction until
-     * we're close to running out of cache.
+     * thread can have a couple of dirty pages before eviction threads kick in. On the other side,
+     * the eviction update and dirty triggers are 90%, so application threads aren't involved in
```
Comment (sueloverso) [inline test/csuite/timestamp_abort/main.c:455]: I have a minor concern that having the `90%` here in the comment but the value way up at the top of the file in the configuration string means they will get out of sync with each other.

File: test/csuite/timestamp_abort/main.c (hunk 1 above)
Comment (ddanderson) [reply test/csuite/timestamp_abort/main.c:111]: I modified `random_abort` and `schema_abort` the same way.

File: test/csuite/timestamp_abort/main.c (hunk 2 above)
Comment (ddanderson) [reply test/csuite/timestamp_abort/main.c]: I've restructured the comments a bit.

**Comment by ddanderson** [comment]
@sueloverso, can you please take another look at this?

**Pull request merged**